### PR TITLE
Fix/content readthedocs2

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
Second PR with content changes for readthedocs

### CODE REVIEW

**Review by santiagoinfantinom — COMMENTED** [review]
Last review of changes. Ready for your approval and a subsequent merging

File: docs/Configuration.rst [hunk 1]
```diff
@@ -106,9 +106,9 @@ In general, the solver of oemof is set to cbc (**solver**). The solver output (*
 
 To increase computation speed (especially for nonconvex flows in the dispatch OEM), an additional solver option **cmdline_option** is added  with value _cmdline_option_value_. It influences, when the solver accepts the found solution as optimal. Possible options:
 
-* ratioGap: ?
-* allowedGap: ?
-* mipgap: ?
+* ratioGap
+* allowedGap
+* mipgap
```
Comment (smartie2076) [inline docs/Configuration.rst:111]:
```suggestion
```

**Review by smartie2076 — APPROVED** [review]
One suggested change

**Review by smartie2076 — COMMENTED** [review]
Actually, you still need to update the changelog!

**Comment by santiagoinfantinom** [comment]
@smartie2076 this should be ready to merge now

**Pull request merged**